spikes/spine-adopt/wazero-probe/compare.go

```
@@ -0,0 +1,414 @@
+type checks struct{ Rows []checkRow }
+
+func (c *checks) check(name string, ok bool, detail string) {
+	r := checkRow{Name: name, Passed: ok}
+	if !ok {
+		r.Detail = detail
+	}
+	c.Rows = append(c.Rows, r)
+}
+
+func (c *checks) eq(name string, got, want any) {
+	g, _ := json.Marshal(got)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)